Jellyfin.Api/Auth/FirstTimeSetupPolicy/FirstTimeSetupHandler.cs

```diff
@@ -28,6 +29,10 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             {
                 context.Succeed(requirement);
             }
+            else if (context.User.GetIsApiKey())
```

Comment:
Should be all admins and not just api key
```suggestion
            else if (context.User.IsInRole(UserRoles.Administrator))
```